akka-http/build.sbt

```diff
@@ -35,6 +35,7 @@ val `akka-http-server` =
       publishSettings,
       `scala 2.12 to dotty`,
       name := "akka-http-server",
+      version := "2.0.0",
```

Comment:
Why are you being specific here with the version?

Reply:
We need to bump the major version number because the changes on the akka-http server interpreter are not backward binary compatible with the previous release.

The algebra stays with a 1.x version because it is binary compatible.